### PR TITLE
#229 | Apply Data Model Changes for Admin User Register Application Service

### PR DESCRIPTION
**Pull request by ibrahimdenizz**
### Description

I made the following changes:
- I renamed `AYS_ADMIN_USER_REGISTER_VERIFICATION` table to `AYS_ADMIN_USER_REGISTER_APPLICATION`.
- I updated the constraints' names according to the new table name.
- I fixed the name of the status column constraint.
- I updated the `verification` keyword to `application` from the entire project.
### Related Issue

#229 

### Proposed Changes

Please provide a brief and descriptive title for your pull request, summarizing the changes or features introduced. It
should be in English and clearly indicate the purpose of the pull request. Optionally, you can provide a more detailed
description of what you have done.

- Branches should follow the below `Naming Conventions of Branchs` header in [README.md](https://github.com/afet-yonetim-sistemi/ays-be/edit/main/README.md) File

### Additional Information

If there's any additional information or context that the reviewers should know, please provide it here.

### Checklist

Please check the following before submitting your pull request:

- [X] I have tested my changes locally and they are working as expected.
- [X] The code is formatted according to the project's coding guidelines and style.
- [X] Necessary documentation has been added or existing documentation has been updated.
- [X] Relevant unit tests have been written and included.
- [X] Relevant integration tests have been written and included.
- [X] The code has been reviewed to ensure its quality.
- [X] The code does not contain any issues flagged by SonarLint.
- [X] Default reviewers have been assigned to this pull request.
- [X] Assignees have been added if necessary.
- [X] Labels have been applied if necessary.
- [X] If an issue is opened and this pull request is related to that issue, it should not be associated with any project
  board or milestone. However, if there is no issue, the pull request should be associated with the appropriate project
  board (BE Board, In Review column) and milestone to track the progress.

### Reviewers

The default project maintainers and team members will be automatically assigned as reviewers for this pull request.

### CODE REVIEW

File: src/test/java/com/ays/admin_user/service/impl/AdminUserRegisterServiceImplTest.java
Comment (agitrubard) [inline src/test/java/com/ays/admin_user/service/impl/AdminUserRegisterServiceImplTest.java:1]: Test sınıfını neden sildik? Özel bir sebebi var mı?

File: src/main/resources/db/changelog/changes/6-ays-dml.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+
+            ALTER TABLE AYS_ADMIN_USER_REGISTER_APPLICATION DROP CONSTRAINT C__AYS_ADMIN_USER_REGISTER_VERIFICATION__ADMIN_USER_ID;
+            ALTER TABLE AYS_ADMIN_USER_REGISTER_APPLICATION
+                ADD CONSTRAINT C__AYS_ADMIN_USER_REGISTER_APPLICATION__STATUS CHECK (STATUS IN ('WAITING', 'COMPLETED', 'VERIFIED', 'REJECTED'));
+
+            UPDATE AYS_ADMIN_USER_REGISTER_APPLICATION
+            SET INSTITUTION_ID = '77ece256-bf0e-4bbe-801d-173083f8bdcf'
+                WHERE id = '3cfeb994-88f4-48d0-b105-309f42f91412';
+        </sql>
```
Comment (agitrubard) [inline src/main/resources/db/changelog/changes/6-ays-dml.xml:37]: Buradaki scriptleri Liquibase'in kullandığı xml formatında yazamıyor muyuz?

**Comment by agitrubard** [comment]
@ibrahimdenizz Selamlar,
Pull request ve issue'yu ilişkilendirebilir miyiz?

**Review by agitrubard — CHANGES_REQUESTED** [review]

File: src/test/java/com/ays/admin_user/service/impl/AdminUserRegisterServiceImplTest.java
Comment (ibrahimdenizz) [reply src/test/java/com/ays/admin_user/service/impl/AdminUserRegisterServiceImplTest.java:1]: @agitrubard Test sınıfı silinmedi sadece çok fazla class ve field ismi değiştirildiği için `Load diff` yazısı çıkıyor.

Comment (agitrubard) [reply src/test/java/com/ays/admin_user/service/impl/AdminUserRegisterServiceImplTest.java:1]: @ibrahimdenizz tamamdır gözümden kaçmış, tekrar inceliyorum

File: src/main/resources/db/changelog/changes/6-ays-dml.xml (hunk 1 above)
Comment (ibrahimdenizz) [reply src/main/resources/db/changelog/changes/6-ays-dml.xml:41]: @agitrubard check constraint komutları pro versiyonunda olduğu için onu sql olarak bıraktım.

Comment (agitrubard) [reply src/main/resources/db/changelog/changes/6-ays-dml.xml:41]: Eline sağlık 🙌🏼

**Review by moaydogdu — APPROVED** [review]
LGTM!

**Review by agitrubard — APPROVED** [review]

**Pull request merged**